R/dataIBDMDBdemo.R

```diff
@@ -0,0 +1,52 @@
+#' IBDMDB 2-omic demo (MGX + MTX)
```

Comment:
Also include original data references in the data documentation. You can check the other demo data sets for examples.